### DISCUSSION TITLE
Why does MarchingCubesLookupTables.cs derive from MonoBehaviour?

### DISCUSSION THREAD

**Discussion by Vastlee** [Eldemarkki/Marching-Cubes-Terrain#54] [Q&A]
See Title

**Comment by Eldemarkki** [comment]
I’m not quite sure how that has slipped in when I implemented it. It is not supposed to derive from MonoBehaviour. I fixed it last week, but it’s only on the develop branch currently. I’m not sure when I’ll merge develop into master, but in the meanwhile you can just remove the MonoBehaviour base class from it. Also, you should mark the class static because I don’t think it currently is (except on develop branch)
